staticcheck.conf

```diff
@@ -1,2 +0,0 @@
-# Temporary: "-SA1019"
```

Comment:
It was deleted because it was the old static-analysis configuration and we are using a different one now?